packages/express/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@clerk/express",
+  "version": "0.0.1",
```

Comment:
I'd publish at 1.0.0 and not in the zero range. Semver is weird in the 0 range and I don't think we need to keep it in that range since the API seems to be safe and sound.